### PR TITLE
Added DPCPP example for GPU SVM

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

**Comment by Alexander-Makaryev** [comment]
@PetrovKP We should add the check is GPU available or not like here
https://github.com/IntelPython/daal4py/blob/master/examples/sycl/covariance_batch.py#L34

**Comment by Alexander-Makaryev** [comment]
@PetrovKP SVM is not released in DAAL yet, we should specify version limitation:
https://github.com/IntelPython/daal4py/blob/master/examples/run_examples.py#L95

**Comment by PetrovKP** [comment]
@Alexander-Makaryev Thanks a lot! Please check, did I do the right?

File: setup.py [hunk 1]
```diff
@@ -94,14 +94,14 @@
...
 
-header_path = os.path.join(daal_root, 'include', 'services', 'library_version_info.h') 
+header_path = os.path.join(daal_root, 'include', 'services', 'library_version_info.h')
 
 with open(header_path) as header:
     v = parse_version(header)
     dal_build_version = (int(v[0]), int(v[2]))
 
 if dpcpp:
-    DPCPP_CFLAGS = ['-D_DPCPP_', '-DONEAPI_DAAL_USE_MKL_GPU_FUNC']
```
[the remark this answers is not in the document]
Comment (PetrovKP) [reply setup.py:104]: For building with oneMKL until beta04. Then we redid the logic. And now the default behavior takes oneMKL.

[the remark this answers is not in the document]
Comment (napetrov) [reply setup.py:104]: ok

**Pull request merged**